api/seedgenerator/v1alpha1/types.go

```diff
@@ -43,10 +43,16 @@ type SeedGenerator struct {
 // SeedGeneratorSpec defines the desired state of SeedGenerator
 type SeedGeneratorSpec struct {
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Seed Image",xDescriptors={"urn:alm:descriptor:com.tectonic.ui:text"}
+	// +kubebuilder:validation:MinLength=1
+	// +kubebuilder:validation:MaxLength=255
```

Comment:
Why 255?

Reply:
yea, nice question... I'd just put it since it is the max number of characters for a full domain name (kinda of standard for DNS systems), but I'm totally Ok if we decide to remove it, don't think etcd has such limitation either...

Reply:
Looks like Podman will limit you to 255:

```
$ podman pull aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa.com/aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa/aaaaaaaaaaaaaaaaaaaa

Error: repository name must not be more than 255 characters
```

Reply:
But the `@sha256:` portion doesn't count towards it:

```
podman pull aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa.com/aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa/aaaaaaaaaaaaaaaaaaa@sha256:80e5e113a984d74836aa16f5b4524012099436b1a50df293f00ac6377fb512c8
```

Reply:
So let's do 350 just to be safe